docs/specs/tripgo.openapi.yaml

```diff
@@ -391,27 +388,39 @@ paths:
                   properties:
                     id:
                       type: string
-                      description: Route ID.
+                      description: ID of the route
+                    operatorID:
+                      type: string
+                      description: ID of the operator servicing this route
                     name:
                       type: string
-                      description: Route name.
+                      description: Long name of the route, if available
+                    shortName:
```

Comment:
This is called `serviceNumber` in https://github.com/skedgo/tripgo-api/blob/fa9818c00e1f49cdd8c8dc99ac4561e9a384c7f8/docs/specs/tripgo.swagger.yaml#L3489. Just wondering if we should call this `number` to be consistent or only change the description for a better reference to the other endpoints.

Reply:
I prefer to adopt `shortName` and deprecate the "number" usage in the other fields. Very often, this is not a number.

Reply:
Ok, sure. Let's move to this name then.